# The custom msg file created by isy on 10/11/2017
# it is used to publish the target positions for 3 robots for 
# formation control and obstacle avoidance using navigation function
Header header
geometry_msgs/Point robot1
geometry_msgs/Point robot2
geometry_msgs/Point robot3
